pyasic/miners/backends/braiins_os.py

```diff
@@ -1058,3 +1062,27 @@ async def _get_uptime(self, rpc_summary: dict = None) -> Optional[int]:
                 return int(rpc_summary["SUMMARY"][0]["Elapsed"])
             except LookupError:
                 pass
+
+    async def _get_pools(self, grpc_pool_groups: dict = None) -> List[PoolMetrics]:
+        if grpc_pool_groups is None:
+            try:
+                grpc_pool_groups = await self.web.get_pool_groups()
+            except APIError:
+                return []
+        pools_data = []
+        for group in grpc_pool_groups:
+            for pool_info in group.pools:
+                pool_data = PoolMetrics(
+                    url=pool_info.url,
+                    user=pool_info.user,
+                    index=int(pool_info.uid),
```

Comment:
This cannot be mapped to an int.  Use `for idx, pool_info in enumerate(group["pools"]`.

Here is an example output - 
```
{'poolGroups': [{'name': 'Default', 'quota': {'value': 1}, 'pools': [{'uid': 'seXGXjvgZn5GvG48', 'url': 'stratum+tcp://realpool.io:3333', 'user': 'MyUser', 'enabled': True, 'alive': True, 'active': True, 'stats': {'acceptedShares': 105, 'lastDifficulty': 262144, 'bestShare': 262144, 'generatedWork': 47982720}}]}]}
```